search_descriptions_raw.yaml

```diff
@@ -14,6 +14,8 @@ company_search_description:
     'liquidation' : "Liquidation"
     'administration' : "In Administration"
     'registered-externally' : "Registered externally as {external_registration_number}"
+    'registered' : "Registered on {date:date_of_creation}"
+    'removed': "Removed on {date:date_of_cessation}"
```

Comment:
Should this have a space before the : on line 18 to be consistently formatted?

Reply:
Good spot. Will fix.